process/main.py

```diff
@@ -521,21 +521,44 @@ def validate_input(self):
                     variables = line.strip().split(sep, 1)[0]
                     variables_in_in_dat.append(variables)
 
-        obs_vars = []
-        replace_hints = []
+        obs_vars_in_in_dat = []
+        replace_hints = {}
         for var in variables_in_in_dat:
-            new_var = obsolete_variables.get(var)
-            if new_var:
-                obs_vars.append(var)
-                replace_hints.append(new_var)
-
-        if len(replace_hints) > 0:
-            print(
-                "The IN.DAT file contains obsolete variables from the OBS_VARS dictionary."
-            )
-            print(
-                f"The obsolete variables in your IN.DAT file are: {obs_vars}. Please change them to the following corresponding new variable(s) before continuing: {replace_hints}."
+            if var in obsolete_variables:
+                obs_vars_in_in_dat.append(var)
+                new_var = obsolete_variables.get(var)
+                replace_hints[var] = new_var
+
+        if len(obs_vars_in_in_dat) > 0:
+            message = (
+                "The IN.DAT file contains obsolete variables from the OBS_VARS dictionary. The obsolete variables in your IN.DAT file are: "
+                f"{obs_vars_in_in_dat}. "
+                "Either remove these or replace them with their updated variable names. "
             )
+            for var in obs_vars_in_in_dat:
+                if replace_hints[var][0] is None:
+                    message = (
+                        message
+                        + "\n \n "
+                        + var
+                        + " is an obsolete variable and does not have a replacement. "
+                    )
```

Comment:
Could probably be a one-line f-string

```suggestion
                    message += f"\n\n {var} is an obsolete variable and does not have a replacement."
```

Also the message feels a bit clunky. Maybe you could say "... is an obsolete variable and should be removed"